src/components/product/Product.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+import InputSpan from "../form/InputSpan";
+
+import './product.scss';
+
+class Product extends Component {
+
+    constructor(props) {
+
+        super(props);
+
+        this.fields = [
+            { label: 'title', reg: /^.{3,}$/ },
+            { label: 'price', reg: /^[0-9]+$/ },
+            { label: 'description', reg: /^.{3,}$/ },
+        ];
+
+        this.state = {};
+
+        this.fields.forEach(({ label }) => this.state[label] = ({
+            value: '',
+            error: false,
+            display: false,
+        }));
+    }
+
+    validate = ({ target }) => {
+        const field = this.fields.find(item => item.label === target.name);
+        const stateField = this.state[target.name];
+
+        if (!field.reg.test(target.value)) {
+            this.setState({
+                [target.name]: {
+                    ...stateField,
+                    error: target.name === 'price' ? "Should be only digits" : "Is not valid"
+                }
+            });
+            return false;
+        } else {
```

Comment:
no need `else` here

Reply:
"else" is needed because we use it on textarea element